### ISSUE TITLE
"policy" key in annotations should be namespaced

### ISSUE THREAD

**Issue by caseydavenport** [projectcalico/calico-kubernetes#13]
Annotations are typically namespaced in Kubernetes.

So,

```
annotations:
  policy: "..."
```

Should become something like 

```
annotations:
  projectcalico.org/policy: "..."
```

**Comment by alexhersh** [comment]
[Code Change](https://github.com/projectcalico/calico-kubernetes/pull/25)
[Doc Change](https://github.com/projectcalico/calico-docker/pull/466)

**Comment by caseydavenport** [comment]
#25 has been merged and will be included in v0.2.0.

The doc change still needs to go in, so I'll leave this open until then.

**Comment by caseydavenport** [comment]
Done as of v0.2.0
